pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=64", "setuptools_scm>=8", "cmake", "swig"]
+requires = ["setuptools>=64", "setuptools_scm>=8", "cmake", "swig", "numpy<2"]
```

Comment:
Would it go here or in the  `dependencies` section below?

Reply:
sorry done